data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json

```diff
@@ -0,0 +1,224 @@
+      },
+      {
+        "text": "Anything on your mind?",
+        "condition": { "u_has_var": "gold_mission_complete", "type": "general", "context": "robofac_merc_1", "value": "yes" },
+        "topic": "TALK_ROBOFAC_MERC_1_RANDOM_THOUGHTS"
+      },
+      { "text": "What do you know about our employers?", "topic": "TALK_ROBOFAC_MERC_1_ASK_HUB" },
+      { "text": "Well, I should get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_ASK_HUB",
+    "type": "talk_topic",
+    "dynamic_line": "Not much really.  They made me track and destroy some government documents a while back, of course I gave them a read first but there wasn't much.\n\n As far as I know, this place held some experimental computer network before everything went to shit.  Whats going on now is anybody's guess.",
+    "responses": [ { "text": "Well, I was hoping for more.", "topic": "TALK_ROBOFAC_MERC_1_MAIN" } ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_GOING_ON",
+    "type": "talk_topic",
+    "dynamic_line": [ "Just came back from a job.  Trying to rest for a short while." ],
+    "responses": [ { "text": "...", "topic": "TALK_ROBOFAC_MERC_1_MAIN" } ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_COINS",
+    "type": "talk_topic",
+    "dynamic_line": "Have you noticed they pay us in tiny little gold coins, its bizarre I say.  One would think they have a gold mine in there, but no, they don't, so they make me cross the dammed world grabbing gold bars. ",
+    "responses": [
+      { "text": "I guess I could help with that...", "topic": "TALK_ROBOFAC_MERC_1_GOLD_MISSION" },
+      { "text": "Now that you mention it, it does seem rather strange.", "topic": "TALK_ROBOFAC_MERC_1_MAIN" }
+    ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_RANDOM_THOUGHTS",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "Thinking I should go hunt something soon...",
+      "Wondering if things will get better someday...",
+      "Hmm? Nothing, I guess I just like resting in this place.",
+      "Have you ever noticed how... wait no, never mind.",
+      "I heard some strange dimensional resonance caused all this, do you think it has happened to other places?",
+      "You noticed this place has working WI-FI?  Well not the rest of the net works anyways."
+    ],
+    "responses": [ { "text": "...", "topic": "TALK_ROBOFAC_MERC_1_MAIN" } ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_GOLD_MISSION",
+    "type": "talk_topic",
+    "dynamic_line": "I can offer you the whole of my payment for every bar or equivalent you bring.  But keep silent about this, its supposed to be my job.",
+    "responses": [
+      { "text": "We have a deal then", "effect": { "add_mission": "MISSION_ROBOFAC_MERC_1_GOLD" }, "topic": "TALK_DONE" },
+      { "text": "Actually, I don't think I have time for that, sorry.", "topic": "TALK_ROBOFAC_MERC_1_MAIN" }
+    ]
+  },
+  {
+    "id": "MISSION_ROBOFAC_MERC_1_GOLD",
+    "type": "mission_definition",
+    "name": "Retrieve gold bar",
```

Comment:
We're not looking specifically for a gold bar, right? Maybe `Retrieve chunks of gold`?